core/src/main/java/org/apache/carbondata/core/datastore/impl/btree/BTreeDataRefNodeFinder.java

```diff
@@ -51,16 +51,15 @@
   /**
    * this will be used during search for no dictionary column
    */
-  private int numberOfNoDictionaryColumns;
+  private int numberOfNoDictSortColumns;
```

Comment:
can you change the comment for this member also. explain what this stored in this member